File: fix_class.py
import os, re

files = []
for root, dirs, filenames in os.walk('C:/Users/bdademo/.qclaw/workspace/edu-platform'):
    for f in filenames:
        if f.endswith('.tsx') or f.endswith('.ts'):
            files.append(os.path.join(root, f))

for fpath in files:
    try:
        with open(fpath, 'r', encoding='utf-8') as f:
            content = f.read()
        new = re.sub(r'\bclass=', 'className=', content)
        if new != content:
            with open(fpath, 'w', encoding='utf-8') as f:
                f.write(new)
            print(f'Fixed: {fpath}')
    except Exception as e:
        print(f'Error {fpath}: {e}')

File: fix_class.js
const fs = require('fs');
const path = require('path');

const root = 'C:/Users/bdademo/.qclaw/workspace/edu-platform';
const files = [];

function walk(dir) {
  for (const entry of fs.readdirSync(dir, { withFileTypes: true })) {
    const full = path.join(dir, entry.name);
    if (entry.isDirectory()) walk(full);
    else if (full.endsWith('.tsx') || full.endsWith('.ts')) files.push(full);
  }
}
walk(root);

let count = 0;
for (const fpath of files) {
  try {
    let content = fs.readFileSync(fpath, 'utf-8');
    const re = /\bclass=/g;
    if (re.test(content)) {
      content = content.replace(re, 'className=');
      fs.writeFileSync(fpath, content, 'utf-8');
      console.log('Fixed:', fpath);
      count++;
    }
  } catch (e) {
    console.error('Error:', fpath, e.message);
  }
}
console.log('Done. Files fixed:', count);
